extensions/notification/destination/KafkaNotificationDestination.js

```diff
@@ -1,5 +1,42 @@
 const NotificationDestination = require('./NotificationDestination');
 const KafkaProducer = require('./KafkaProducer');
+const { ZenkoMetrics } = require('arsenal').metrics;
+
+const notificationsSent = ZenkoMetrics.createCounter({
+    name: 'notification_queue_processor_notifications_sent',
```

Comment:
"sent" is kind of redundant : bucket notificaiton is only about sending notifications.

maybe it would be clearer with 'count' in the name, to more explicitely describe what this is about:
`notification_queue_processor_notifications_sent`

Reply:
The metric was _removed_ ?
Counting the notifications (to show the number of notifications per seconds) seems quite useful though, my earlier comment was really just about the _name_ of the metric.

Reply:
We can use the `deliveryLag` metric to do the same, It's a summary so we can just get the count